<html>
<head>

  <meta name="navigation" content="Comparative Genomics">
  <title>Ancestral sequences</title>


</head>

<body>

<h1>Ancestral sequences</h1>

<p>
<a href="https://europepmc.org/articles/PMC2577868">Ancestral sequences</a> are inferred from the
<a href="/info/genome/compara/multiple_genome_alignments.html">EPO multiple alignments</a> using Ortheus.
Ortheus is a probabilistic method for the inference of ancestor, a.k.a tree, alignments. The main
contribution of Ortheus is the use of a phylogenetic model incorporating gaps to infer insertion
and deletion events. Ancestral sequences are predicted for each node of the phylogenetic tree that
relates the sequences. Each ancestral sequence is named according to the derived extant species.
For example, a sequence named "((oryza_sativa,oryza_indica),oryza_rufipogon)" corresponds to the
ancestor of the <i class="taxonomy">Oryza sativa</i> Japonica, <i class="taxonomy">Oryza indica</i>
Indica, and <i class="taxonomy">Oryza rufipogon</i> genomes.
</p>

</body>
</html>