<html>
<head>
<meta name="order" content="4" />
<title>Synteny</title>
</head>

<body>

<h1>Synteny</h1>

<p><a href="https://en.wikipedia.org/wiki/Synteny">Synteny</a> is the
  conserved order of aligned genomic blocks between species. Ensembl
  Plants includes some synteny analyses generated in house, and some
  generated by our collaborators <a href="https://gramene.org/">Gramene</a>.
  These methods are summarised below.
</p>

<p>Information on current Ensembl Plants synteny data is listed alongside
  <a href="/info/genome/compara/compara_analyses.html">available alignments</a>.
</p>

<h2>Ensembl synteny method</h2>

<p>In-house synteny analyses are calculated from <a href="/info/genome/compara/compara_analyses.html">
  pairwise genome alignments</a>, when both species have a chromosome-level assembly.
</p>

<p>This is done in two phases:</p>

<ol>
  <li>We search for alignment blocks that are in the same order in the two genomes.
    Syntenic alignments that are closer than 200 kb are grouped into a synteny block.
  </li>
  <li>Groups that are in synteny are linked, provided that no more than two non-syntenic
    groups are found between them and they are less than 3 Mb apart.
  </li>
</ol>

<p>To avoid overly sparse syntenic views, a synteny dataset may be excluded
  if it has average genomic coverage below a threshold, typically 5%.
</p>

<p>For some synteny datasets, analysis parameters may have been tuned to account for differences
  in the underlying data such as genome size and evolutionary distance.
</p>

<h2>Gramene synteny method</h2>

<p>The Gramene method is based on <a href="/info/genome/compara/peptide_compara.html">orthology</a>.
  For each species combination, the orthologues are assigned a gene index (1..last) depending on
  order along the chromosome (non-orthologous genes are skipped). DAGchainer software is used to
  detect collinear genes contained in syntenic blocks, and the coordinates of the syntenic block
  are derived from the outermost genes within each block. There is no post-processing, so syntenic
  blocks can sometimes overlap or be nested. The method was used in
  <a href="https://europepmc.org/abstract/MED/19965430">The B73 Maize Genome: Complexity, Diversity,
  and Dynamics</a>. Schnable PS <i>et al</i>. 2009. Science 326(5956):1112-1115.
</p>

</body>
</html>
